spec/acceptance/5_client_spec.rb

```diff
@@ -11,29 +11,49 @@ class { 'keycloak':
       keycloak_realm { 'test': ensure => 'present' }
       keycloak_flow { 'foo on test': ensure => 'present' }
       keycloak_client { 'test.foo.bar':
-        realm                          => 'test',
-        root_url                       => 'https://test.foo.bar',
-        redirect_uris                  => ['https://test.foo.bar/test1'],
-        default_client_scopes          => ['address'],
-        secret                         => 'foobar',
-        login_theme                    => 'keycloak',
-        backchannel_logout_url         => 'https://test.foo.bar/logout',
-        authorization_services_enabled => false,
-        service_accounts_enabled       => true,
-        browser_flow                   => 'foo',
-        roles                          => ['bar_role', 'other_bar_role'],
+        realm                                   => 'test',
```

Comment:
For both additions in this file, it might be better to create a new `keycloak_client` that uses SAML.  The one that was in this file was to test OIDC.

Reply:
Do you this way?